clients/drcachesim/tests/window_test.cpp

```diff
@@ -159,7 +158,7 @@ test_main(int argc, const char *argv[])
         assert(status == scheduler_t::STATUS_OK);
         if (memref.marker.type == TRACE_TYPE_MARKER &&
             memref.marker.marker_type == TRACE_MARKER_TYPE_TIMESTAMP) {
-            // Looks at the gap between the current and the previous timestamps
+            // Looks at the gap between the first and second timestamps
```

Comment:
Hmm, should it check all of them (i.e., remove the break)?

Reply:
That equates to checking that there's no sleep between any two trace timestamps, which is not true.

OTOH there _could_ be a sleep between the first two trace timestamps too. I feel this test is inherently brittle and may be working on because the -trace_after_instrs/-retrace_every_instrs/-trace_for_instrs settings are amenable.

Reply:
Opened #6847. Going ahead with this PR and will tackle these existing issues in #6847.